src/inc/redirect/Mlp_Language_Negotiation.php

```diff
@@ -109,10 +110,11 @@ private function get_possible_matches( Array $translations ) {
 	private function get_fallback_match() {
 
 		return array (
-			'priority' => 0,
-			'url'      => '',
-			'language' => '',
-			'site_id'  => 0
+			'priority'          => 0,
+			'url'               => '',
+			'language'          => '',
+			'site_id'           => 0,
+			'target_content_id' => 0,
```

Comment:
First of all, thanks for you feedback!

What do you think of `post_id` instead?